views/admin/grading.html

```diff
@@ -1,7 +1,7 @@
+{{ response.title="Grading" }}
```

Comment:
Should this be removed in favor of the PR I just accepted that sets this in the controller?

Reply:
Yeah, I don't know how that snuck in there.

Reply:
Removed from the latest commit.